### ISSUE TITLE
Training Failure

### ISSUE THREAD

**Issue by golddohyun** [zhshi0816/Video-Frame-Interpolation-Transformer#14]
Hi, I got interested in implementing your paper. But the problem is that... - I have tried to execute your code, and got the result below. I used python 3.8, torch 1.12.1, cuda 11.3, A100 as torch 1.5 and cuda 10 environment is not available for me. I set batch size 8 and trained from Vimeo-90k septuplets dataset.

Here's my training log below..
Is there any way to train in a higher version of the cuda environment?

![image](https://user-images.githubusercontent.com/104754233/218597172-aa52d47e-d9d4-4809-a396-b524b8a571b1.png)

**Comment by FR6024** [comment]
Have you already fix it? I meet the same probleme

**Comment by FR6024** [comment]
> Hi, I got interested in implementing your paper. But the problem is that... - I have tried to execute your code, and got the result below. I used python 3.8, torch 1.12.1, cuda 11.3, A100 as torch 1.5 and cuda 10 environment is not available for me. I set batch size 8 and trained from Vimeo-90k septuplets dataset.
> 
> Here's my training log below.. Is there any way to train in a higher version of the cuda environment?
> 
> ![image](https://user-images.githubusercontent.com/104754233/218597172-aa52d47e-d9d4-4809-a396-b524b8a571b1.png)

Have you already fix it?

**Comment by golddohyun** [comment]
> Have you already fix it? I meet the same probleme

No.. still suffering with the same problem :(

**Comment by dzambrano** [comment]
hello @golddohyun, I'm also interested in this problem. I have a question: your problem is that the loss doesn't decrease? What makes you believe that the reason is in the higher cuda version?

**Comment by dzambrano** [comment]
Hello, I just wanted to report that one reason for not training with the newer libs versions might be the data augmentation:
indeed, as reported [here](https://pytorch.org/vision/stable/transforms.html), since torchvision==0.8.0, you need to add `torch.manual_seed(seed)` to [this line](https://github.com/zhshi0816/Video-Frame-Interpolation-Transformer/blob/2b1dc91f2da617ce1a7ea924262dc498e52a91cf/dataset/vimeo90k_septuplet.py#L71) to make sure all the input images get the same augmentation.

**Comment by esthen-bit** [comment]
This problem is caused by the `random_seed` function in `vimeo90k_septuplet` , replace the `random.seed(seed)` with function below and you will figure it out.
```
def set_seed(seed=None, cuda=False): 
    random.seed(seed)
    os.environ['PYTHONHASHSEED'] = str(seed)
    np.random.seed(seed)
    torch.manual_seed(seed)
    torch.cuda.manual_seed(seed)
    if cuda:
        torch.cuda.manual_seed_all(seed)
```
